src/tox_lsr/test_scripts/runcommitlint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Do not exit on an error to continue ansible-doc and ansible-test.
+set -euo pipefail
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+
+pushd "$LSR_TOX_ENV_DIR"
+npm install @commitlint/config-conventional @commitlint/cli
```

Comment:
Oh yes, that's a little shame that this must be done manually because commitlint is Java. I've added a subsection `Additional requirements` and info about commitlint to it. I noticed that shellcheck package is required too, so I've added info about it to the same section.
I've also added a check for `npm` presense to runcommitlint.sh.